sympy/combinatorics/permutations.py

```diff
@@ -3002,7 +3002,7 @@ class AppliedPermutation(Expr):
     >>> _.subs(x, 1)
     2
     """
-    def __new__(cls, perm, x, evaluate=global_evaluate[0]):
+    def __new__(cls, perm, x, evaluate=global_parameters.evaluate):
```

Comment:
This was incorrect and should be fixed. Default values for function parameters are evaluated at the time of function definition not function call. I guess this isn't tested.